source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -36,40 +36,44 @@ let initialize_configuration
         _;
       }
   =
-  Log.info "Verifying model syntax and configuration.";
+  Log.info "Verifying taint configuration.";
+  let open Core.Result in
+  TaintConfiguration.from_taint_model_paths taint_model_paths
+  >>= TaintConfiguration.with_command_line_options
+        ~rule_filter
+        ~source_filter
+        ~sink_filter
+        ~transform_filter
+        ~find_missing_flows
+        ~dump_model_query_results_path:dump_model_query_results
+        ~maximum_model_source_tree_width
+        ~maximum_model_sink_tree_width
+        ~maximum_model_tito_tree_width
+        ~maximum_tree_depth_after_widening
+        ~maximum_return_access_path_width
+        ~maximum_return_access_path_depth_after_widening
+        ~maximum_tito_collapse_depth
+        ~maximum_tito_positions
+        ~maximum_overrides_to_analyze
+        ~maximum_trace_length
+        ~maximum_tito_depth
+  |> TaintConfiguration.exception_on_error
+
+
+let initialize_configuration_with_models ~static_analysis_configuration =
```

Comment:
I would avoid this pattern of `function foo; function foo_with_bar`. Just make two separate functions, one for taint config and one for model syntax check. Use `Statistics.performance` in both with different (phase) names.

Reply:
okay!